docs/command_line_tool/phylum_extension_install.md

```diff
@@ -4,29 +4,32 @@ category: 6255e67693d5200013b1fa3e
 hidden: false
 ---
 
-Install an extension
+Install extension
 
 ```sh
-phylum extension install [OPTIONS] <PATH>
+Usage: phylum extension install [OPTIONS] <PATH>
 ```
 
+### Arguments
+
+<PATH>
+
 ### Options
-`--accept-permissions`
-&emsp; Automatically accept requested permissions
 
-`--overwrite`
-&emsp; Overwrite existing extension
+-y, --yes
+&emsp; Accept permissions and overwrite existing extensions (same as --overwrite --accept-permissions)
 
-`-y`, `--yes`
-&emsp; Accept permissions and overwrite existing (same as `--overwrite --accept-permissions`)
+--accept-permissions
+&emsp; Automatically accept requested permissions
 
-### Details
+--overwrite
+&emsp; Overwrite existing extension
 
-The extension will be installed under `$XDG_DATA_HOME/phylum/extensions/<EXT_NAME>`.
-If `$XDG_DATA_HOME` is not set, it will default to `$HOME/.local/share/phylum/extensions/<EXT_NAME>`.
```

Comment:
These details were dropped. Was that on purpose? It might be worth adding them back in at the command definition so they show up in the CLI help and docs.

Reply:
Good catch, I only looked for example sections, I wasn't aware of this one.